lib/include/public/Version.hpp

```diff
@@ -6,8 +6,8 @@
 #define MAT_VERSION_HPP
 // WARNING: DO NOT MODIFY THIS FILE!
 // This file has been automatically generated, manual changes will be lost.
-#define BUILD_VERSION_STR "3.8.249.1"
```

Comment:
You can revert this change unless we need a new release. I think we will need a new release but it will be after the Java/Obj-C/Swift changes. So for now we just leave version.hpp as is.